src/styles/blocks/_buttons.scss

```diff
@@ -0,0 +1,109 @@
+.button-cont {
+  
+
+  &--shop {
+    @include onTablet {
+      display: flex;
+      justify-content: center;
+      margin-top: 40px;
+    }
+
+    @include onDesktop {
+      display: flex;
+      justify-content: center;
+      margin-top: 40px;
+    }
+  }
+
+  
+}
+
+.button {
+  width: 100%;
+  margin-bottom: 30px;
+  height: 60px;
+  font-size: 16px;
+  line-height: 21px;
+  text-align: center;
+  background-color: $color-black;
+  color: $color-gray2;
+  text-transform: uppercase;
+  cursor: pointer;
+
+  &:hover {
+    background-color: $color-gray2;
+    color: $color-black;
+    outline: none;
+    border: 1px solid $color-black;
+  }
+
+  &__link {
+    color: $color-gray2;
+    text-transform: uppercase;
+    &:hover {
+      // display: block;
+      color: $color-black;
+    }
+  }
+
+  @include onDesktop {
+    width: 35%;
+    margin: 0 0 48px;
+  }
+
+  // @include onTablet {
+  //   width: 40%;
+  //   margin: 0 auto;
+  // }
+
+  &--shop {
+    @include onTablet {
+      width: 50%;
+    }
+  }
+
+  &--creators {
+    @include onDesktop {
+      width: 100%;
+      margin-top: 60px;
+    }
+
+    @include onTablet {
+      width: 100%;
+      margin: 40px 0 0;
+    }
+  }
+}
+
+.lern {
+  display: inline-block;
+  margin-top: 40px;
+
+  &__link-cont--powder {
+    position: relative;
+    top: 400px;
+
+    @include onTablet {
+      top: 250px;
+    }
+
+    @include onDesktop {
+      top: 270px;
+    }
+  }
+
+  &__link {
+    font-family: Sharp Sans, sans-serif;
+    font-size: 16px;
+    font-weight: 500;
+    line-height: 18px;
+    letter-spacing: 1px;
+    color: $color-green2;
+    padding-bottom: 10px;
+    transition: border-bottom 1s;
```

Comment:
1 second is too long for transition, optimal variant would be 0.3 seconds, also make sure that transition speed is the same everywhere, make a variable for it